powertools-metrics/src/main/java/software/amazon/lambda/powertools/metrics/PowertoolsMetricsLogger.java

```diff
@@ -0,0 +1,29 @@
+package software.amazon.lambda.powertools.metrics;
+
+import java.util.function.Consumer;
+
+import software.amazon.cloudwatchlogs.emf.logger.MetricsLogger;
+import software.amazon.cloudwatchlogs.emf.model.Unit;
+
+public final class PowertoolsMetricsLogger {
+    private static final MetricsLogger metricsLogger = new MetricsLogger();
+
+    public static MetricsLogger metricsLogger() {
+        return metricsLogger;
+    }
+
+    public static void withSingleMetric(final String name,
```

Comment:
Did you leave the service out on purpose?

Reply:
Yeah made the API consistent with python one here https://awslabs.github.io/aws-lambda-powertools-python/core/metrics/#creating-a-metric-with-a-different-dimension